piccolo_admin/endpoints.py

```diff
@@ -774,7 +790,7 @@ async def store_file(
 
         try:
             file_key = await media_storage.store_file(
-                file_name=file.filename,
+                file_name=file.filename,  # type: ignore
```

Comment:
What's the error with this?

Reply:
There was a mypy error for this, but now I tried to clone the branch into a fresh virtualenv, ran linter and the error went away. Sorry, it's my fault. I will remove it.

Reply:
No problem - just thought it might be something I needed to fix.